doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,282 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
+                    echo 'Fail, but I can recover, try recovery.'; false;
+                else
+                    echo 'Fail, I will never figure this out!'; false;
+                fi
+            fi
+        """
+
+    [[recovery]]
+        command scripting = """
+            sleep 10;
+            if (($RANDOM % 2)); then
+                echo 'Success, I could be helped by the recovery task'; true;
+            else
+                echo 'Fail, I could not be helped by the recovery task'; false;
+            fi
+       """
+</pre>
+    </div>
+
+      <p>and a <samp>suite-gotoend.rc</samp> file that looks like this:</p>
+
+    <div class="slide">
+        <h3 class="alwayshidden">Advanced example suite-gotoend.rc</h3>
+<pre class="prettyprint lang-cylc">
+# If a member of a family fails carry on.
+# If a specified task fails go to the end of that cycle and remove the failed
+# task from from the task pool.
+# suite-gotoend.rc
+
+[scheduling]
+    [[dependencies]]
+        graph = """
+            start_install                => sometimes_fail => FAMILY_PASS
+            FAMILY_PASS:succeed-all      => SOME_DO_SOME_DONT
+            SOME_DO_SOME_DONT:finish-all => dependent_on_families
+            sometimes_fail:fail          => !sometimes_fail &\
+            !FAMILY_PASS & !SOME_DO_SOME_DONT & !dependent_on_families
+            dependent_on_families | sometimes_fail:fail => housekeep
+        """
+
+[runtime]
+    [[FAMILY_PASS]]
+        command scripting = sleep 10;
+
+    [[SOME_DO_SOME_DONT]]
+        command scripting = sleep 5;
+
+    [[bar]]
+        inherit = FAMILY_PASS
+        command scripting = echo 'bar always succeeds'
+
+    [[dependent_on_families]]
+       command scripting = """
+           sleep 10;
+           echo 'I can only run if all FAMILY_PASS succeed and SOME_DO_SOME_DONT finish'
+           """
+
+    [[foo]]
+        inherit = FAMILY_PASS
+        command scripting = echo 'foo always succeeds'
+                           
+    [[flaky_member]]
+        inherit = SOME_DO_SOME_DONT
+        command scripting = """
+            echo 'flaky member is going to:'
+            sleep 10;
+            if (($RANDOM % 2)); then
+                echo 'Success'; true; 
+            else
+                echo 'Fail'; false;
+            fi
+        """
+    [[sometimes_fail]]
+       command scripting = """
+           sleep 10;
+           if (($RANDOM % 2)); then
+               echo 'Success'; true;
+           else
+               echo 'Fail'; false;
+           fi
+       """
+
+    [[stable_member]]
+        inherit = SOME_DO_SOME_DONT
+        command scripting = echo 'stable member always succeeds'
+</pre>
+    </div>
+
+      <p>and a <samp>suite-twofail.rc</samp> file that looks like this:</p>
+
+    <div class="slide">
+        <h3 class="alwayshidden">Advanced example suite-twofail.rc</h3>
+<pre class="prettyprint lang-cylc">
+# If the first task fails got the end of the cycle or if two specified task fail
```

Comment:
Could you be a bit more specific in what "first task" etc are here - since they're in a file here they're not abstract so should be named for clarity when looked at in isolation from the documentation page. Same applies to the other include files.